src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ChangeToolStripParentVerb.cs

```diff
@@ -28,7 +28,7 @@ internal ChangeToolStripParentVerb(string text, ToolStripDesigner designer)
         _designer = designer;
         _provider = designer.Component.Site;
         _host = (IDesignerHost)_provider.GetService(typeof(IDesignerHost));
```

Comment:
I believe this API is available to make this simplification:
```suggestion
        _host = _provider.GetService<IDesignerHost>();
```
The same change can be applied below.